src/components/molecules/MenuPopoverContent/MenuPopoverContent.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+
+import {
+  HOMEPAGE_URL,
+  PRIVACY_POLICY,
+  TERMS_AND_CONDITIONS_URL,
+} from "settings";
+
+import { getExtraLinkProps } from "utils/url";
+
+import "./MenuPopoverContent.scss";
+
+export const MenuPopoverContent: React.FC = () => (
+  <div className="MenuPopoverContent">
+    <a
+      href={TERMS_AND_CONDITIONS_URL}
+      {...getExtraLinkProps(true)}
+      className="MenuPopoverContent__link"
```

Comment:
let's always move `...extra` type of props to the end. (1 / 3)
```suggestion
      className="MenuPopoverContent__link"
      {...getExtraLinkProps(true)}
```

Reply:
Not always.

You move the most important ones to the end so they can possibly overwrite existing properties added before.

In this case it will not make much difference because `getExtraLinkProps()` returns disjunct set from the ones used in the element

Reply:
@sunny-viktoryia @goran-peoski-work 
have to agree with Goran here, it sure is a good practice to use spread/destructuring in the end, but there are cases when you want to override props that come from `...obj`, probably would be a good idea to mark those with some comments or w/e

Reply:
What I do in these cases is to pluck that one prop out in the props, then just reuse it. Sure it adds more unnecessary code, but at least it's visible outside the `...rest` gathered ones. Example

```tsx
      <Link
        className={parentClasses}
        style={style}
        to={disabled ? "#" : linkTo}
        {...getExtraLinkProps(newTab && !disabled)}
        title={title}
        onClick={() => onClick?.()}
        {...extraProps}
      >
```

In the above code taken from `ButtonNG`, the `title` prop is never changed, it just exists as a sort of pass-through variable that adds few lines of code to be more explicit with the intent. But, as always YMMV